index.js

```diff
@@ -45,6 +46,34 @@ function filterResponseForImportantEvents(allEventsFromFetch) {
     return arrayOfImportantEvents;
 }
 
+function shouldIncludeEvent(eventType, association = 'OWNER') {
+    if (eventType.author_association !== association) {
+        return true;
+    }
+
+    return false;
+}
+
+function filterByAuthorAssociation(events) {
+    const filteredEvents = events.filter((event) => {
+        switch (event.type) {
+            case 'PullRequestEvent':
+            case 'PullRequestReviewEvent':
+                return shouldIncludeEvent(event.payload.pull_request);
+            case 'CommitCommentEvent':
+            case 'IssueCommentEvent':
+            case 'PullRequestReviewCommentEvent':
+                return shouldIncludeEvent(event.payload.comment);
+            case 'IssuesEvent':
+                return shouldIncludeEvent(event.payload.issue);
+            default:
+                return [];
```

Comment:
This should either return true, or false. I suspect it should be false, but double check that.